### PR TITLE
Add CommunityList skeleton component

### PR DESCRIPTION
**Pull request by Oksamies**
refs TS-1822

### CODE REVIEW

**Comment by Oksamies** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #989** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/989?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/989?utm_source=stack-comment).

File: packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  flex-direction: column;
+  gap: var(--space--12);
+
+  --skeleton-background: #191b38;
+  --skeleton-border-radius: var(--border-radius--4);
+}
+
+.image {
+  border-radius: var(--skeleton-border-radius);
+  object-fit: cover;
```
Comment (anttimaki) [inline packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css:12]: There's probably no objects we need to fit?

File: packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  background: var(--skeleton-background);
+  aspect-ratio: 3 / 4;
+}
+
+.footer {
```
Comment (anttimaki) [inline packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css:17]: This seems unused?

**Review by anttimaki — APPROVED** [review]
You might want to clean up some unnecessary CSS, but otherwise seems good to go 👍🏻

**Comment by Oksamies** [comment]
### Merge activity

* **Jan 3, 9:16 AM**: @Oksamies started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/989).

**Pull request merged**